filebrowser/run.sh

```diff
@@ -21,10 +21,21 @@ if [ "${DB_PATH}" != "filebrowser.db" ]; then
   DB_FLAG=" -d ${DB_PATH}"
 fi
 
+# Check if filebrowser db exists
+if [ ! -f ${DB_PATH} ]; then
+  filebrowser $DB_FLAG config init >> ${LOG_PATH} 2>&1
+  filebrowser $DB_FLAG users add admin "" --perm.admin=true --viewMode=mosaic >> ${LOG_PATH} 2>&1
+fi
+
+filebrowser $DB_FLAG config set --baseurl=${SERVER_BASE_PATH} --port=${PORT} --auth.method=noauth --root=$ROOT_DIR >> ${LOG_PATH} 2>&1
+
+printf "👷 Starting filebrowser in background... \n\n"
+
+
 printf "📂 Serving $${ROOT_DIR} at http://localhost:${PORT} \n\n"
 
-printf "Running 'filebrowser --noauth --root $ROOT_DIR --port ${PORT}$${DB_FLAG} --baseurl ${SERVER_BASE_PATH}' \n\n"
+printf "Running 'filebrowser $DB_FLAG' \n\n"
```

Comment:
```suggestion
printf "Running 'filebrowser %s'\n\n" "$DB_FLAG"
```
Since we're using printf, this is the typical way to include variable content. 😄

Reply:
Again following convention, changes made